CMakeLists.txt

```diff
@@ -369,6 +363,7 @@ if(SYMFORCE_BUILD_SYMENGINE)
                -DCMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES}
                -DCMAKE_POLICY_DEFAULT_CMP0074=NEW
                -DCMAKE_PREFIX_PATH=${EXTERNAL_PREFIX_PATH}
+    UPDATE_COMMAND ${CMAKE_COMMAND} -E echo "No update step"
```

Comment:
Yeah this makes sense because these are vendored dependencies with SOURCE_DIR, I'm actually not even sure what the default update step would do here.  This part seems good, I'm a little confused by the other change.